openspec/changes/dashboard-shell/tasks.md

```
@@ -0,0 +1,49 @@
+## 1. Dashboard shell layout
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)